css/index.css

```diff
@@ -112,6 +141,88 @@ p {
     justify-content: center;
 }
 
+/* Card styling */
+/* I've used Flexbox here to neatly organize the cards.
+I used flexbox because each card contains 4  */
+
+#card-parent {
+    display: flex;
+    flex-flow: row wrap;
+    margin: 10px 0px;
+    justify-content: space-around;
+    box-sizing: border-box;
+    width: 100%;
+}
+
+.card {
+    display: flex;
+    flex-direction: column;
+    width: 40%;
+    padding-bottom: 20px;
+    margin: 10px;
+    border-bottom: 1px dashed black;
+    margin: 20px auto;
+    flex-basis: auto;
+    flex: 1 .5 auto;
+}
+
+.card img {
+    width: 90%;
+}
+
+.card p {
+    width: 90%;
+    margin: 2% 0;
+}
+
+.card-p {
+    padding: 5px 10px;
+    border: 2px solid black;
+    border-radius: 5px;
+}
+
+/* Adding color to the boxes */
+
+#box-1 {
```

Comment:
great job targeting each box without re-writing the box styles over and over again